dashboard/app/controllers/reference_guides_controller.rb

```diff
@@ -1,11 +1,16 @@
 class ReferenceGuidesController < ApplicationController
   include CurriculumHelper
   before_action :find_reference_guide, only: [:show]
-  before_action :require_levelbuilder_mode_or_test_env, except: [:show]
+  before_action :find_reference_guides, only: [:index]
+  before_action :require_levelbuilder_mode_or_test_env, except: [:show, :index]
```

Comment:
So I think this page is meant to be for levelbuilders to edit all the reference guides for a course version. So it should require levelbuilder mode right?